server/lib/nl/constants.py

```diff
@@ -155,10 +155,75 @@
 
 # TODO: remove this special casing when a better NER model is identified which
 # can always detect these.
-SPECIAL_PLACES: Set[str] = {'palo alto', 'mountain view'}
+SPECIAL_PLACES: Set[str] = {
+    'palo alto', 'mountain view', 'world', 'earth', 'african', 'asian',
+    'european', 'north american', 'south american', 'latin american', 'oceania',
+    'american'
+}
 
 SPECIAL_PLACE_REPLACEMENTS: Dict[str, str] = {'us': 'United States'}
 
+SPECIAL_PLACES_TO_GEOIDS: Dict[str, str] = {
```

Comment:
Is it worth calling it OVERRIDE_FOR_MAPS_API ?

Reply:
updated